kolibri/core/assets/src/state/actions.js

```diff
@@ -233,11 +233,11 @@ function getCurrentSession(store, force = false) {
     });
 }
 
-function getFacilityConfig(store) {
+function getFacilityConfig(store, facilityId = '') {
```

Comment:
You shouldn't actually set a default empty string here. Letting `facilityId` be undefined when nothing is passed in is more semantically correct, because in theory the empty string could be an ID.

Reply:
alternatively you can set it to `null` by default